awscli/examples/s3api/get-bucket-policy.rst

```diff
@@ -1,14 +1,9 @@
-Get and put a bucket policy
----------------------------
+The following command retrieves the bucket policy for a bucket named ``my-bucket``::
 
-The following example shows how you can download an Amazon S3 bucket policy,
-make modifications to the file, and then use ``put-bucket-policy`` to
-apply the modified bucket policy.  To download the bucket policy to a file,
-you can run::
+  aws s3api get-bucket-policy --bucket my-bucket
 
-  aws s3api get-bucket-policy --bucket mybucket --query Policy --output text > policy.json
```

Comment:
I think we should keep the original version.  It shows how you can get the actual JSON policy saved to a file rather than the "JSON in JSON" string you get in the output.